airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml

```diff
@@ -66,6 +66,14 @@ connectionSpecification:
               examples:
                 - refresh_token
               airbyte_secret: true
+            access_token:
```

Comment:
this is not implemented in OAuth flow, I think we need to stay consistent

Reply:
Fixed